model/SyncLog/SyncLogService.php

```diff
@@ -147,4 +150,17 @@ public function search(SyncLogFilter $filter)
     {
         return $this->getStorage()->search($filter);
     }
+
+    /**
+     * @param SyncLogEntity $entity
+     * @param $entityId
+     * @throws \ReflectionException
+     */
+    private function setEntityId(SyncLogEntity $entity, $entityId)
```

Comment:
What is the purpose of this function? Isn't a setter would be more straightforward to use?

Reply:
Entity ID is kind of immutable property, therefore I didn't create setter method. When we create new entity in database I want to update entity object with ID returned by storage implementation